parse/wasm/wasm.go

```diff
@@ -8,11 +8,11 @@ import (
 	"fmt"
 	"syscall/js"
 
-	"github.com/kwilteam/kwil-db/kuneiform"
+	"github.com/kwilteam/kwil-db/parse"
```

Comment:
Up a few lines, on the `//go:generate` line, there appears to be a typo, lacking a space between `netgo` and `wasm.go` (it's passing the tag `netgowasm.go` and the source is implied).

```diff
diff --git a/parse/wasm/wasm.go b/parse/wasm/wasm.go
index d3ba98ce..19413e1e 100644
--- a/parse/wasm/wasm.go
+++ b/parse/wasm/wasm.go
@@ -1,6 +1,6 @@
 //go:build js && wasm
 
-//go:generate env GOOS=js GOARCH=wasm go build -o kuneiform.wasm -ldflags "-s -w" -trimpath -tags netgowasm.go
+//go:generate env GOOS=js GOARCH=wasm go build -o kuneiform.wasm -ldflags "-s -w" -trimpath -tags netgo wasm.go
 package main
```

Probably good to add ` CGO_ENABLED=0` too.